src/wallet/coincontrol.h

```diff
@@ -19,12 +21,12 @@ class CCoinControl
     bool fAllowOtherInputs;
     //! Includes watch only addresses which match the ISMINE_WATCH_SOLVABLE criteria
     bool fAllowWatchOnly;
-    //! Override estimated feerate
+    //! Override automatic min/max checks on fee, m_feerate must be set if true
     bool fOverrideFeeRate;
```

Comment:
In commit "Refactor to use CoinControl in GetMinimumFee and FeeBumper"

You should definitely get rid of the `fOverrideFeeRate` member now that `m_feerate` is optional.